src/rust/src/lib.rs

```diff
@@ -68,7 +68,7 @@ mod rust_bridge {
         pub protocol_version: u32,
         pub sequence_number: u32,
         pub timestamp: u64,
-        pub network_passphrase: Vec<u8>,
+        pub network_id: Vec<u8>,
```

Comment:
Should this type be `[u8; 32]`?

Reply:
Probably, but I don't want to mess with the bridge code to pass arrays.

Reply:
Issue opened related to this - https://github.com/stellar/rs-soroban-env/issues/671.